tasks/db_deploy/migrations/migrate_versions_1_to_2/migrate_sql.py

```diff
@@ -17,7 +17,7 @@ def dbo_role_sql(db_name: str, admin_username: str) -> text:
     Returns:
         SQL for creating orca_dbo role.
     """
-    return text(
+    return text(  # nosec
```

Comment:
curious to know what error it throws without # nosec

Reply:
Since we use `f"""` it says that we are vulnerable to SQL injection.